src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageEndpoint.cs

```diff
@@ -172,6 +173,11 @@ public async Task<RazorMapToDocumentRangesResponse> Handle(RazorMapToDocumentRan
                 };
             }
 
+            if (documentSnapshot is null)
```

Comment:
`null` responses resulted in lost messages, due to out-of-sync message types between RZLS and VS.HTMLCSharp